src/main/java/com/alex/eshop/eshop/Repository/ItemRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.alex.eshop.eshop.Repository;
+
+import com.alex.eshop.eshop.Entity.Item;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.awt.print.Pageable;
+import java.util.List;
+
+@Repository
+public interface ItemRepository extends JpaRepository<Item, Long> {
+    @Query("select i from Item i")
+    List<Item> getLastFiveItems(Pageable pageable);
+
+    List<Item> getItemsByCategory_Id(Long categoryId);
```

Comment:
use camel-case for method names.

P.S. 
There is one interesting thing - method name can de simplified to `getByCategoryId` or `findByCategoryId`. Between `get/find` and `By` parts you can add any text.